site/en/install/pip.html

```diff
@@ -32,7 +32,8 @@ <h3>System requirements</h3>
   <li>macOS 10.12.6 (Sierra) or later (64-bit) <em>(no GPU support)</em></li>
   <li>Windows 7 or later (64-bit) <em>(Python 3 only)</em></li>
```

Comment:
Perhaps add as a bullet point under Windows.
We already do note this in the Windows tab selector.

Reply:
Just fixed that, is this right? (plz forgive my poor Html code 🤦‍♂

Reply:
Almost! Webmaster-in-training 😄

Reply:
Wow, thanks a lot! 👏